app/controllers/posts_controller.rb

```diff
@@ -32,6 +32,11 @@ def create
     end
   end
 
+  def destroy
+    @user_id = params[:user_id]
+    Post.find_by(id: params[:post_id]).destroy
+  end
+
   private
 
   def post_params
```

Comment:
- [ ] Good job working on this but please kindly ensure the PostController uses CanCanCan for Post authorization by including this `load_and_authorize_resource` at the before the methods. Please also ensure that everything still works fine after this.